flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/NtTable.java

```diff
@@ -0,0 +1,91 @@
+package com.flash3388.frc.nt;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class NtTable {
+
+    private final NetworkTable mTable;
+
+    public NtTable(NetworkTable table) {
+        mTable = table;
+    }
+
+    public NtTable(String name) {
+        this(NetworkTableInstance.getDefault(), name);
+    }
+
+    public NtTable(NetworkTableInstance tableInstance, String name) {
+         this(tableInstance.getTable(name));
+    }
+
+    public boolean getBooleanEntryValue(String name) {
```

Comment:
You don't actually need to say "entry". Shorter names can be nice.

Reply:
Wanted to be safe, but whateves